src/coreclr/vm/amd64/gmsamd64.cpp

```diff
@@ -22,7 +22,7 @@ void LazyMachState::unwindLazyState(LazyMachState* baseState,
     }
     CONTRACTL_END;
 
-    CONTEXT                         ctx;
+    CONTEXT                         ctx = { 0 };
```

Comment:
This is a real one. It leads to uninitialized use of `ContextFlags` here:

https://github.com/dotnet/runtime/blob/a1c40287acea351b0e6419cacc468d8af637f387/src/coreclr/pal/src/exception/seh-unwind.cpp#L353

Reply:
Good catch!

Reply:
We have the same code for arm64 and arm. Does it need to be fixed there as well?

Also, `CONTEXT` is a pretty big structure. Would it better to just set the `ContextFlags` that are causing the problem?

Reply:
I agree with both comments, I am sorry for not realizing it before.